src/mapping.ts

```diff
@@ -58,6 +59,16 @@ export function handleTrade(event: Trade): void {
   buyToken.totalVolume =  tokenCurrentBuyAmount.plus(buyAmount)
 
   trades.getOrCreateTrade(event, buyToken, sellToken)
+
+
+  sellToken.priceUSD = getPrice(sellTokenAddress).get("usd")
+  buyToken.priceUSD = getPrice(buyTokenAddress).get("usd")
+  sellToken.priceETH = getPrice(sellTokenAddress).get("eth")
+  buyToken.priceETH = getPrice(buyTokenAddress).get("eth")
```

Comment:
Rather than calling `getPrice` for each twice, wouldn't it be better to store it in a variable?
Unless this is a pattern on TheGraph/WASM I'm not aware